tests/integration/requirements.txt

```diff
@@ -1,5 +1,9 @@
 pyVim
-pyVmomi>=6.7
 podman
 requests
 pycdlib
+ansible-core
+
+# see the cluster_settings role README.md for an explanation on the <7.0.3 restriction
+pyVmomi>=6.7,<7.0.3
```

Comment:
Not sure that's fine to do that... 
What is the reason for that limitation?

Reply:
https://github.com/ansible-collections/community.vmware/pull/2080